nextcord/client.py

```diff
@@ -1873,6 +1872,40 @@ def add_view(self, view: View, *, message_id: Optional[int] = None) -> None:
 
         self._connection.store_view(view, message_id)
 
+    def remove_view(self, view: View, message_id: Optional[int] = None) -> None:
+        """Removes a :class:`~nextcord.ui.View` from persistent listening.
+
+        This method should be used if a persistent view is set in a cog and
+        should be freed when the cog is unloaded to save memory.
+
+        .. versionadded:: 2.3
+
+        Parameters
+        ----------
+        view: :class:`nextcord.ui.View`
+            The view to remove from dispatching.
+        message_id: Optional[:class:`int`]
+            The message ID that the view is attached to. This is used to properly
+            remove the view from the view store.
+
+        Raises
+        ------
+        TypeError
+            A view was not passed.
+        ValueError
+            The view is not persistent. A persistent view has no timeout
+            and all their components have an explicitly provided custom_id.
+        """
+        if not isinstance(view, View):
+            raise TypeError(f"Expected an instance of View not {view.__class__!r}")
```

Comment:
```suggestion
            raise TypeError(f"Expected an instance of View not {view.__class__.__name__}")
```
This is inconsistent with the rest of the library